gen_import_utils.py

```diff
@@ -47,6 +47,27 @@ def replace_slashes_in_dict(json):
         elif isinstance(value, dict):
             replace_slashes_in_dict(value)
 
+def extract_last_folders(path, number: int):
+    """truncates a path string to keep only the last n elements of a path"""
+    path_components = path.split('/')
+    return '/'.join(path_components[-number:])
+
```

Comment:
parses picturae image folder structure to use as paths variable in client_tools.